testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorSmokeTest.java

```diff
@@ -234,8 +234,7 @@ public void testDeleteAllDataFromTable()
     public void testRowLevelDelete()
     {
         skipTestUnless(hasBehavior(SUPPORTS_CREATE_TABLE) && hasBehavior(SUPPORTS_ROW_LEVEL_DELETE));
-        // TODO (https://github.com/trinodb/trino/issues/5901) Use longer table name once Oracle version is updated
-        try (TestTable table = new TestTable(getQueryRunner()::execute, "test_row_delete", "AS SELECT * FROM region")) {
+        try (TestTable table = new TestTable(getQueryRunner()::execute, "test_row_level_delete_" + randomNameSuffix(), "AS SELECT * FROM region")) {
```

Comment:
`TestTable` adds suffix internally.  Please remove `randomNameSuffix`.

Reply:
thanks for your reminder!